legal-api/src/legal_api/core/meta/filing.py

```diff
@@ -480,7 +480,28 @@ def get_all_outputs(business_type: str, filing_name: str) -> list:
         return []
 
     @staticmethod
-    def alter_outputs(filing: FilingStorage, outputs: set):
+    def is_special_resolution_correction(
```

Comment:
This needs to be refactored somewhere else, it's used multiple times in the project. - next fix I'll give it a shot